core/modules/filter/filter.api.php

```diff
@@ -134,28 +134,32 @@ function hook_filter_info_alter(&$info) {
  *   which should be unique and therefore prefixed with the name of the module.
  *   Each value is an associative array describing the editor, with the
  *   following elements (all are optional except as noted):
- *   - title: (required) A human readable name for the editor.
+ *   - label: (required) A human readable name for the editor.
  *   - settings callback: The name of a function that returns configuration
  *     form elements for the editor. See hook_editor_EDITOR_settings() for
  *     details.
  *   - default settings: An associative array containing default settings for
  *     the editor, to be applied when the editor has not been configured yet.
+ *   - file: The name of a file containing the editor settings callback.
+ *   - library: An associative array containing additional libraries.
  *   - js settings callback: The name of a function that returns configuration
  *     options that should be added to the page via JavaScript for use on the
  *     client side. See hook_editor_EDITOR_js_settings() for details.
  *
- * @see filter_example.module
```

Comment:
The `filter_example.module` uses the `hook_filter_info()` which is totally different than `hook_editor_info()`.